commercial/conf/routes

```diff
@@ -3,49 +3,49 @@
+GET        /_healthcheck                                                        conf.HealthCheck.healthCheck()
 
 # Travel offer merchandising components
-GET        /commercial/travel/offers.json                                                       controllers.commercial.TravelOffersController.renderTravel
+GET        /commercial/travel/offers.json                                       controllers.commercial.TravelOffersController.renderTravel
 
 # Job merchandising components
-GET        /commercial/jobs.json                                                                controllers.commercial.JobsController.renderJobs
+GET        /commercial/jobs.json                                                controllers.commercial.JobsController.renderJobs
 
 # Masterclass merchandising components
-GET        /commercial/masterclasses.json                                                       controllers.commercial.MasterclassesController.renderMasterclasses
+GET        /commercial/masterclasses.json                                       controllers.commercial.MasterclassesController.renderMasterclasses
 
 # Soulmates merchandising components
-GET        /commercial/soulmates/$subgroup<\w+>.json                                            controllers.commercial.SoulmatesController.renderSoulmates(subgroup)
-GET        /commercial/soulmates-test/$subgroup<\w+>.json                                       controllers.commercial.SoulmatesController.renderSoulmatesTest(subgroup)
+GET        /commercial/soulmates/$subgroup<\w+>.json                            controllers.commercial.SoulmatesController.renderSoulmates(subgroup)
+GET        /commercial/soulmates-test/$subgroup<\w+>.json                       controllers.commercial.SoulmatesController.renderSoulmatesTest(subgroup)
 
 # Money merchandising components
-GET        /commercial/money/bestbuys.json                                                      controllers.commercial.MoneyOffers.renderBestBuys
-GET        /commercial/money/savings/:savingsType                                               controllers.commercial.MoneyOffers.savings(savingsType)
-GET        /commercial/money/current-accounts/:currentAccountType                               controllers.commercial.MoneyOffers.currentAccounts(currentAccountType)
-GET        /commercial/money/credit-cards/:creditCardType                                       controllers.commercial.MoneyOffers.creditCards(creditCardType)
-GET        /commercial/money/loans/:loanType                                                    controllers.commercial.MoneyOffers.loans(loanType)
+GET        /commercial/money/bestbuys.json                                      controllers.commercial.MoneyOffers.renderBestBuys
+GET        /commercial/money/savings/:savingsType                               controllers.commercial.MoneyOffers.savings(savingsType)
```

Comment:
It's a parameter you pass to the method:
https://www.playframework.com/documentation/2.5.x/JavaRouting#The-routes-file-syntax